server/index.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+import dotenv from 'dotenv';
```

Comment:
I tried running the dev script, and it said there was no dotenv. I had to install dotenv in the package.json. We should probably add that.